docs/docs/reference/crd-reference/app.md

```diff
@@ -54,60 +45,37 @@ when the app discovery feature generates the `KeptnApp` resource:
-    For example, increment this number to restart a `KeptnApp` version
-    that failed to deploy, perhaps because a
-    `preDeploymentEvaluation` or `preDeploymentTask` failed.
-    See
-    [Restart an Application Deployment](../../guides/restart-application-deployment.md)
-    for a longer discussion of this.
+       The value is an integer that can be modified
+       to trigger another deployment of a `KeptnApp` of the same version.
+       For example, increment this number to restart a `KeptnApp` version
+       that failed to deploy, perhaps because a
+       `preDeploymentEvaluation` or `preDeploymentTask` failed.
+       See
+       [Restart an Application Deployment](../../guides/restart-application-deployment.md)
+       for a longer discussion of this.
   - **workloads**
     - **name** (required) -- name of this Kubernetes
-      [workload](https://kubernetes.io/docs/concepts/workloads/).
-      Use the same naming rules listed above for the application name.
-      Provide one entry for each [workload](https://kubernetes.io/docs/concepts/workloads/)
-      associated with this Keptn application.
+            [workload](https://kubernetes.io/docs/concepts/workloads/).
+            Use the same naming rules listed above for the application name.
+            Provide one entry for each [workload](https://kubernetes.io/docs/concepts/workloads/)
+            associated with this Keptn application.
     - **version** (required) -- version number for this [workload](https://kubernetes.io/docs/concepts/workloads/).
-      Changing this number causes a new execution
-      of checks for this [workload](https://kubernetes.io/docs/concepts/workloads/) only,
-      not the entire application.
-
-The remaining fields are required only when implementing
-the release lifecycle management feature.
-If used, these fields must be populated manually:
-
-- **preDeploymentTasks** -- list each task
-    to be run as part of the pre-deployment stage.
-    Task names must match the value of the `metadata.name` field
-    for the associated [KeptnTaskDefinition](taskdefinition.md) resource.
-- **postDeploymentTasks** -- list each task
-    to be run as part of the post-deployment stage.
-    Task names must match the value of the `metadata.name` field
-    for the associated
-    [KeptnTaskDefinition](taskdefinition.md)
-    resource.
-- **preDeploymentEvaluations** -- list each evaluation to be run
-    as part of the pre-deployment stage.
-    Evaluation names must match the value of the `metadata.name` field
-    for the associated
-    [KeptnEvaluationDefinition](evaluationdefinition.md)
-    resource.
-- **postDeploymentEvaluations** -- list each evaluation to be run
-    as part of the post-deployment stage.
-    Evaluation names must match the value of the `metadata.name` field
-    for the associated [KeptnEvaluationDefinition](evaluationdefinition.md)
-    resource.
+            Changing this number causes a new execution
```

Comment:
users should not change this anymore. Maybe we can leave out this information

Reply:
done